### PR TITLE
APP-458 Operations Metrics

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Collect operations metrics using Metrics library from Dropwizard

### CODE REVIEW

**Comment by robson-sanchez** [comment]
@dnsymphony please check.

File: integration-metrics/src/main/java/org/symphonyoss/integration/metrics/IntegrationMetricsController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * Controller class to delegate the metric objects creation to the specific controller.
+ * Created by rsanchez on 12/12/16.
+ */
+@Component
+public class IntegrationMetricsController {
+
+  /**
+   * Holds the list of controller to be initialized
+   */
+  private List<IntegrationController> controllers = new ArrayList<>();
```
Comment (dnsymphony) [inline integration-metrics/src/main/java/org/symphonyoss/integration/metrics/IntegrationMetricsController.java]: You could also autowire this collection.  Spring will initialize the collection with all beans that implement IntegrationController interface.  Then controllers wouldn't need to self register.  Downside to autowiring is that new controllers cannot be registered after container is initialized.

Comment (robson-sanchez) [reply integration-metrics/src/main/java/org/symphonyoss/integration/metrics/IntegrationMetricsController.java]: Nice tip :+1:

FIXED

**Comment by robson-sanchez** [comment]
@dnsymphony please check again.

**Review by dnsymphony — APPROVED** [review]

**Pull request merged**